include/multipass/cli/client_common.h

```diff
@@ -53,6 +53,7 @@ std::unique_ptr<SSLCertProvider> get_cert_provider();
 void set_logger();
 void set_logger(multipass::logging::Level verbosity); // full param qualification makes sure msvc is happy
 void preliminary_setup();
+void concluding_setup();
```

Comment:
Having slept on it a bit… how about this?
```suggestion
void final_setup();
```

Reply:
I thought concluding was more precise that final: this setup concludes execution, rather than being the last setup. As suggested in IRC, another option could be `prefix_setup` and `postfix_setup`. A bit more change, but probably makes the symmetry clearer for the reader. Let me know.

Reply:
Going with `pre_setup` and `post_setup` (as requested in IRC).

Reply:
Done